src/compiler/types/lines_0_1000.rs

```diff
@@ -548,6 +586,10 @@ pub trait NodeInterface: ReadonlyTextRange {
     fn maybe_locals(&self) -> RefMut<Option<SymbolTable>>;
     fn locals(&self) -> RefMut<SymbolTable>;
     fn set_locals(&self, locals: Option<SymbolTable>);
+    fn maybe_js_doc(&self) -> Option<Vec<Rc<Node /*JSDoc*/>>>;
```

Comment:
These only apply to `Node` types that also `extend JSDocContainer` but seems to make sense to put them on `NodeInterface`/`BaseNode`